src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -140,11 +155,31 @@ export default class ItemDetailPage extends React.PureComponent<Props, State> {
                 </div>
               ) : (
                 <div className="metrics">
+                  <div className="subtitle">{t('item_detail_page.properties')}</div>
                   <div className="metric">{t('model_metrics.triangles', { count: metrics.triangles })}</div>
                   <div className="metric">{t('model_metrics.materials', { count: metrics.materials })}</div>
                   <div className="metric">{t('model_metrics.textures', { count: metrics.textures })}</div>
                 </div>
               )}
+              {isSmart(item) ? (
+                <ItemVideo item={item}>
+                  {(_video, duration, size, _isLoading) => (
+                    <>
+                      <div className="overlay">
+                        <Icon name="play" className="play-video-button" onClick={this.handleEditItemVideo} />
+                      </div>
+                      <Button primary onClick={this.handleEditItemVideo}>
```

Comment:
WDYT about opening the modal directly in the upload step when the user clicks on the button and the video preview when clicking on the play icon?